src/test/java/POM/pages/BasePage.java

```diff
@@ -19,6 +19,8 @@ public class BasePage {
     @FindBy(xpath = "//a[contains(text(),'Albums')]")
     WebElement albumsMenuItem;
 
+    @FindBy(xpath = "//a[contains(text(),'Artists')]")
+    WebElement artistsMenuItem;
```

Comment:
**I recommend: make the variables protected**, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

_For example_

```
    protected WebDriver driver;
    protected WebDriverWait wait;
    protected Actions actions;
```


**I recommend:  make all the locators and WebElements are private**, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

_For example_


```
    @FindBy(xpath = "//a[contains(text(),'Songs')]")
    private WebElement allSongsMenuItem;
    @FindBy(xpath = "//a[contains(text(),'Albums')]")
    private WebElement albumsMenuItem;
    @FindBy(xpath = "//a[contains(text(),'Artists')]")
    private WebElement artistsMenuItem;
    @FindBy(xpath = "//a[text()='Favorites']")
    private WebElement favoritesMenuItem;
```